compiler/ballerina-lang/src/main/java/io/ballerina/projects/Module.java

```diff
@@ -99,6 +102,18 @@ public boolean isDefaultModule() {
         return moduleContext.isDefaultModule();
     }
 
+    public Project project() {
+        return this.moduleContext.project();
+    }
+
+    /** Returns an instance of the Module.Modifier.
+     *
+     * @return  module modifier
+     */
+    public Modifier modify() {
```

Comment:
Shall we change the method name to the noun, instead of the verb since this act as a getter?